transaction/migrations/0001_initial.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 2.2.2 on 2019-06-21 09:36
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('company', '0002_auto_20190621_0935'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Transaction',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('date', models.DateField()),
+                ('money', models.IntegerField()),
+                ('type', models.CharField(choices=[('IN', 'income'), ('EX', 'expense')], max_length=2)),
+                ('description', models.TextField()),
```

Comment:
We've called this name for now, do you think description is better?

Reply:
I think description is better, and that is what I used in swagger

Reply:
I think that `name` implies that they are unique in some way, and that it's just a word used to refer to it, not to describe the purpose.